src/field/arena.ts

```diff
@@ -289,8 +289,8 @@ export class Arena {
 
   /**
    * Sets weather to the override specified in overrides.ts
-   * @param weather new weather to set of type WeatherType
-   * @returns true to force trySetWeather to return true
+   * @param weather {@linkcode WeatherType} new weather to set of type WeatherType
+   * @returns boolean - true to force trySetWeather to return true
```

Comment:
```suggestion
   * @returns true to force trySetWeather to return true
```
you don’t need to put types into JS doc as we are using ts. This is just redundant